plugins/vcr.rst

```diff
@@ -0,0 +1,96 @@
+VCR Plugin
+==========
+
+The VCR plugins allow you to record & replay HTTP responses.
+
+Install
+-------
+
+.. code-block:: bash
+
+    $ composer require --dev php-http/vcr-plugin
+
+Usage
+-----
+
+To record or replay a response, you will need two components, a **naming strategy** and a **recorder**.
+
+The naming strategy
+*******************
+
+The naming strategy turn a request into a deterministic and unique identifier.
+The identifier must be safe to use with a filesystem.
+The plugin provide a default naming strategy, the ``PathNamingStrategy``. You can define two options:
+
+* **hash_headers**: the list of header names to hash (Ex: Authorization),
+* **hash_body_methods**: methods for which the body will be hashed (Default: PUT, POST, PATCH)
```

Comment:
similar here, the config tells for which request methods the body makes requests distinct.